Group1:
  signalGen_Amplitude : 1.0
  signalGen_freq: 1.0